src/ansiblelint/config.py

```diff
@@ -60,7 +60,7 @@
 
 
 options = Namespace(
-    cache_dir=None,
+    cache_dir=os.path.expanduser("~/") + ".cache",
```

Comment:
Can this be something like
```suggestion
    cache_dir=pathlib.Path.home() / ".cache",
```
?